src/pages/Main.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box } from '@mui/material'
```

Comment:
move this component out into a new folder `layouts` and you can call this component `DefaultLayout` or `RootLayout` or something like `LoggedInLayout`